sdk/src/lib.rs

```diff
@@ -23,6 +23,7 @@ pub mod epoch_info;
 pub mod exit;
 pub mod feature;
 pub mod feature_set;
+pub mod fees;
```

Comment:
It's not clear me why this lives in the SDK?

Reply:
Only so I wouldn't have to move it when bringing the BanksClient methods in line with these RpcClient ones. I don't feel strongly, though, so will move to `/client` if you prefer.

Reply:
I think I _slightly_ prefer keeping it in `/client`, given how general the mod name of "fees" is.  Or rather, out of `sdk/` for that same reason.  Perhaps it moves into `banks-interface/` eventually if it's only shared between `client/` and `banks-client/`.

Reply:
Moved.